### DISCUSSION TITLE
QuickJS instead of Deno for app backend!

### DISCUSSION THREAD

**Discussion by Himujjal** [tauri-apps/tauri#1131] [Ideas]
Instead of Deno. Can't QuickJS be used as a backend for the app? This is just an idea. Of course this hasn't been implemented yet. But considering size of the app is a good marketing point, QuickJS might just be more beneficial in that regard instead of Deno. And to be frank, the performance is not that bad!

**Comment by nothingismagick** [comment]
We discussed this last night at our Core Team call, and feel like these second layer things should be addressed once we have beta nailed down. At any rate, we want to be inclusionary instead of exclusionary, so in my opinion it should be QuickJS && Deno. :)
